testing/sdk_repository.py

```diff
@@ -96,18 +100,194 @@ def remove_universe_repos(stub_urls):
     log.info('Finished removing universe repos')
 
 
-def universe_session():
+def add_package_registry_stub() -> dict:
+    stub_url = os.environ['PACKAGE_REGISTRY_STUB_URL']
+    with urllib.request.urlopen(stub_url) as url:
```

Comment:
This seems like a very specific check, and I would argue that this is applicable to all universe repos.

Also, is it guaranteed that a stub universe URL only contains a single package?

Reply:
Yes, this was intentional. In long term, we will get rid of this method and install package registry from the cluster. I added a TODO for the same.